docs/connecting-data/connecting-to-apis-and-external-data/clickup-api.md

```diff
@@ -1,7 +1,168 @@
----
-description: How to Connect DataDistillr to the ClickUp API
----
+# Connecting to ClickUp
 
-# ClickUp API
+## Creating a ClickUp Account
+Set up an account with [ClickUp](https://app.clickup.com/signup){target=_blank}.
 
-Details coming soon!
\ No newline at end of file
+???+ cost
+    There are several different types of accounts:
+
+    - Free
+    - Unlimited - $9 per member per month
+    - Business - $19 per member per month
+    - Business Plus - $29 per member per month
+    - Enterprise
+
+[Details and Comparison](https://clickup.com/pricing){target=_blank}
```

Comment:
Include this link in the Cost box - right now it's separated and unclear which section it belongs to